opa/opa.go

```diff
@@ -37,6 +38,7 @@ type client struct {
 	logger       *zap.Logger
 	Host         string
 	ExplainQuery bool
+	HTTPClient   *http.Client
```

Comment:
Nitpick, but the client should probably be a private field.

Reply:
will do!